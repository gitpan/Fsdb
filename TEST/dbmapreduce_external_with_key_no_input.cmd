prog='dbmapreduce'
args='--nowarnings -K -k experiment perl TEST/dbmapreduce_external_with_key.pl'
cmp='diff -cb '
in=TEST/dbmapreduce_no_input.in
