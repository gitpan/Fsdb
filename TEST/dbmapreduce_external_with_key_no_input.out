#fsdb experiment key
-	-
#   | dbmapreduce --nowarnings -K -k experiment perl TEST/dbmapreduce_external_with_key.pl
